eclipse/che
# Remove TS_SELENIUM_MULTIUSER variable.

We don't use/test with singleuser for quite some time now.

Signed-off-by: Radim Hopp <rhopp@redhat.com>

### What does this PR do?
Remove TS_SELENIUM_MULTIUSER variable used in e2e tests. We don't test singleuser for quite some time anymore AFAIK.


### Screenshot/screencast of this PR



### What issues does this PR fix or reference?



### How to test this PR?
Happy path test should test it ;-)
For local execution you can use something like this:
```
podman run --shm-size=512m -v <PATH_TO_LOCAL_TESTS_FOLDER>:/tmp/e2e:Z -e TS_SELENIUM_BASE_URL=<URL_TO_CHE_DEPLOYMENT> -e TS_SELENIUM_OCP_USERNAME=<SUERNAME> -e TS_SELENIUM_OCP_PASSWORD=<PASSWORD> -e TEST_SUITE=test-java-vertx -e NODE_TLS_REJECT_UNAUTHORIZED=0 -e TS_OCP_LOGIN_PAGE_PROVIDER_TITLE=<OCP_IDP_TITLE> -e TS_SELENIUM_VALUE_OPENSHIFT_OAUTH=true quay.io/eclipse/che-e2e:nightly
```


### PR Checklist

[As the author of this Pull Request I made sure that:](https://github.com/eclipse/che/blob/master/CONTRIBUTING.md#pull-request-template-and-its-checklist)


### Reviewers

Reviewers, please comment how you tested the PR when approving it.
